graphqlapi/src/main/java/io/stargate/graphql/GraphqlActivator.java

```diff
@@ -44,6 +44,8 @@
       System.getProperty("stargate.auth_id", "AuthTableBasedService");
   private static final boolean ENABLE_GRAPHQL_FIRST =
       Boolean.getBoolean("stargate.graphql_first.enabled");
+  private static final boolean DISABLE_GRAPHQL_PLAYGROUND =
```

Comment:
nit: would it be worth using `ENABLE` in the name to align with `ENABLE_GRAPHQL_FIRST`?

It would also make the `if` in DropwizardServer.java  a bit more readable IMHO.

Reply:
`Boolean.parseBoolean(System.getProperty("stargate.graphql_playground.enabled", "true"))`

Reply:
I was initially going to use "enable" but it felt weird to have such a flag default to "true". I agree that it makes the `if` more legible but from a user perspective this feels like unexpected behavior. This also aligns with the existing property `stargate.graphql.default_keyspace.disabled`.

Reply:
The default value logic makes sense to me. Could this be a middle ground if we had something like this:
```
boolean ENABLE_GRAPHQL_PLAYGROUND = !Boolean.getBoolean("stargate.graphql_playground.disabled");
```
... and used positive "enable" parameter names in code? WDYT?

Reply:
Done

Reply:
> This also aligns with the existing property `stargate.graphql.default_keyspace.disabled`

Hrmm I didn't remember that property existed, that's unfortunate. FWIW I prefer to avoid double negatives ("not disabled"), and it would be slightly easier if you look at a bunch of properties at once and true means enabled all the time.
But maybe I'm overthinking it, +0.